include/cuco/detail/static_multimap/kernels.cuh

```diff
@@ -416,6 +416,7 @@ __global__ void retrieve(InputIt first,
     idx += loop_stride;
   }
 
+  flushing_cg.sync();
```

Comment:
If the loop doesn't iterate at all, then we never sync the CG before the read on the following line. Additionally, `retrieve` and `retrieve_outer` only sync the `active_flushing_cg` (which might not sync all of `flushing_cg` AIUI).